curriculum/challenges/english/25-front-end-development/workshop-fruit-search-app/67f8eeab71f4f24ba1a3a7eb.md

```diff
@@ -0,0 +1,129 @@
+---
+id: 67f8eeab71f4f24ba1a3a7eb
+title: Step 10
+challengeType: 0
+dashedName: step-10
+---
+
+# --description--
+
+After the form, add a `div` element with an `id` of `results` that will display the search results.
+
+# --hints--
+
+You should have a `div` element with the `id` `results` after the form.
```

Comment:
two words wrapped in backticks cannot be next to each other. see issue I linked in an earlier comment.